manifests/database/postgresql.pp

```diff
@@ -1,15 +1,16 @@
 # Class for creating the PuppetDB postgresql database. See README.md for more
 # information.
 class puppetdb::database::postgresql(
-  $listen_addresses     = $puppetdb::params::database_host,
-  $database_name        = $puppetdb::params::database_name,
-  $database_username    = $puppetdb::params::database_username,
-  $database_password    = $puppetdb::params::database_password,
-  $database_port        = $puppetdb::params::database_port,
-  $manage_database      = $puppetdb::params::manage_database,
-  $manage_server        = $puppetdb::params::manage_dbserver,
-  $manage_package_repo  = $puppetdb::params::manage_pg_repo,
-  $postgres_version     = $puppetdb::params::postgres_version,
+  $listen_addresses       = $puppetdb::params::database_host,
+  $database_name          = $puppetdb::params::database_name,
+  $database_username      = $puppetdb::params::database_username,
+  $database_password      = $puppetdb::params::database_password,
+  $database_port          = $puppetdb::params::database_port,
+  $manage_database        = $puppetdb::params::manage_database,
+  $manage_server          = $puppetdb::params::manage_dbserver,
+  $manage_package_repo    = $puppetdb::params::manage_pg_repo,
+  $postgres_version       = $puppetdb::params::postgres_version,
```

Comment:
Looks like we need one more space on these `=` above, so they line up with the one below.